openslides_backend/action/actions/motion/update.py

```diff
@@ -121,3 +124,54 @@ def set_recommendation_extension_reference_ids(
         instance[
             "recommendation_extension_reference_ids"
         ] = recommendation_extension_reference_ids
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        motion = self.datastore.get(
+            FullQualifiedId(self.model.collection, instance["id"]),
+            ["meeting_id", "state_id", "submitter_ids"],
+        )
+
+        # check for can_manage, all allowed
+        perm = Permissions.Motion.CAN_MANAGE
+        if has_perm(self.datastore, self.user_id, perm, motion["meeting_id"]):
+            return
+
+        # check for can_manage_metadata and whitelist
+        perm = Permissions.Motion.CAN_MANAGE_METADATA
+        whitelist = [
+            "category_id",
+            "motion_block_id",
+            "origin",
+            "supporters_id",
+            "recommendation_extension",
+        ]
+        allowed_fields = ["id"]
+        if has_perm(self.datastore, self.user_id, perm, motion["meeting_id"]):
+            allowed_fields += whitelist
```

Comment:
no need for the `whitelist` variables be up there, just add the lists here

Reply:
Removed the whitelist var here.